File: C++/Student.cpp
#include "Student.hpp"


	//constructor (initial value)
	Student::Student() : User(){
        
    }
	//Constructor (inputed values)
	Student::Student(string first_name, string last_name, string id) : User(first_name, last_name, id){
	}

	void Student::search_course(){
        cout << "Search course for Student!" << endl;
    }
    void Student::add_course(){
        cout << "Add Course for Student!" << endl;
    }
    void Student::drop_course(){
        cout << "Drop Course for Student!" << endl;
    }
    void Student::print_schedule(){
        cout << "Print Schedule for Student!" << endl;
    }

    //Deconstructor
    Student::~Student(){

    }

File: Java/Student.java
public class Student extends User{
    Student(){
        super();
    }

    Student(String first_name, String last_name, String id){
        super(first_name, last_name, id);
    }

    public void search_course(){
        System.out.println("Search Course for Student!");
    }
    public void add_course(){
        System.out.println("Add Course for Student!");
    }
    public void drop_course(){
        System.out.println("Drop Course for Student!");
    }
    public void print_schedule(){
        System.out.println("Print Schedule for Student!");
    }

}
